files/new_user_email_notify/watchnewuser.sh

```diff
@@ -0,0 +1,81 @@
+#!/bin/sh
+# Script that will grep a log file and send an email when a specified pattern is encountered.
+__ScriptName="watchnewuser.sh"
```

Comment:
Ahh, do we have two of these scripts doing more or less the same thing?

Reply:
Probably, will need to look into this further. All I know regarding this set of scripts is that one of for email notification and one is for api notification.